src/renderer/base/lib/base.vcxproj.filters

```diff
@@ -27,6 +27,9 @@
     <ClCompile Include="..\FontInfoDesired.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\FontResource.cpp">
+      <Filter>Source Files</Filter>
+    </ClCompile>

```

Comment:
Note that I added the `FontResource` class to renderer/base, even though it's currently only used by the GDI renderer, because I was hoping it could eventually be reused (with some modification) in the DX renderer too.

Reply:
This'll need to be added to the `sources` file in that directory as well :smile: